.env

```diff
@@ -0,0 +1 @@
+secretkey = "NewNewThings"
```

Comment:
Remove this file from the commit and ignore the file using the `.gitignore` file. This file should be present in your folder but it should not be committed.